modules/_labs/select/react/lib/SelectMenu.tsx

```diff
@@ -258,14 +268,7 @@ const SelectMenu = ({
   }, [handleWidthChange]);
 
   useCloseOnEscape(popupRef, () => onCloseOnEscape?.());
-
-  // TODO: use the useCloseOnOutsideClick hook after it's been modified to handle
-  // multiple things happening on the same click. Currently, the hook doesn't
-  // support the use case where one Select Menu is activated by clicking on its
-  // Select Button while another Select Menu is already active (both Menus are
-  // visible instead of the original Menu dismissing). In order to support the
-  // correct behavior for now, we're dismissing the Menu on blur instead of
-  // using the hook.
+  useCloseOnOutsideClick(popupRef, () => onCloseOnOutsideClick?.());
```

Comment:
I was able to use Popup's `useCloseOnOutsideClick` hook to replace the functionality lost by removing the Menu's blur handler (previously, `useCloseOnOutsideClick` could leave multiple Select Menus open if a page contained multiple Selects -- that issue has since been resolved).

`useCloseOnOutsideClick` also has the additional benefit of making it much easier to inspect the Menu (previously, the Menu would remove itself from the DOM as soon as you clicked anywhere else, including the browser developer tools).